Update docs (#49)
hlesey/k8s-corabia: README.md

<p align="center"><img alt="kind" src="./docs/images/k8s-corabia.png" width="300px" /></p>

# K8s Corabia

> This repository provides helps you to run a minimum best-practice Kubernetes Cluster for educational and testing purpose that can run on your local machine or in the cloud.

Reviewer: ```suggestion
This repository provides help to run a minimum best-practice Kubernetes cluster for educational and testing purposes. The cluster can be run on your local machine or in the cloud.
```
